src/track/track.cpp

```diff
@@ -454,7 +458,7 @@ void Track::emitChangedSignalsForAllMetadata() {
     emit titleChanged(getTitle());
     emit albumChanged(getAlbum());
     emit albumArtistChanged(getAlbumArtist());
-    emit genreChanged(getGenre());
+    emit genreTextChanged(getGenreText());
```

Comment:
Could we split off the genre text renamings and already merge them or is this too much work?

Reply:
This could probably reduce the change set. I will check that.